core/src/main/resources/reference.conf

```diff
@@ -1611,6 +1611,15 @@ datastax-java-driver {
         significant-digits = 3
         refresh-interval = 5 minutes
       }
+
+      # The time after the node level metrics will be evicted.
+      # The eviction will happen only if none of the enabled node-level metrics is updated for a given node for a given time.
+      # When the eviction-time will elapse, all metrics for a given node are removed.
+      #
+      # Required: no (defaults to 1 hour)
+      # Modifiable at runtime: no
+      # Overridable in a profile: no
+      eviction-time = 1 hour
```

Comment:
I wonder if we need to expose this in the configuration. Maybe 1 hour is a reasonable one-size-fit-all value that we could hard-code instead.

We've had that philosophy of "when in doubt, make it configurable" in driver 4, but the config file is getting huge. I think we should be more opinionated sometimes.

Reply:
Regarding one hour, I am not sure if this is `one-size-fit-all`. 
- Maybe there will be some clients that will not tolerate stale metrics for one hour but also tolerate some risk of flickering and set it to N minutes?
- Or on the other hand, maybe someone that uses our driver in a context of batch jobs (i.e. spark) that are executed every N hours will need to increment that parameters to couple hours to retain metrics between those batch job runs?